Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM alpine:edge
+
+RUN echo http://dl-cdn.alpinelinux.org/alpine/edge/testing >> /etc/apk/repositories && \
+    apk add --no-cache uwsgi-python git ctags  py3-six py3-markupsafe py3-pygments \
+                       py3-dulwich py3-humanize py3-flask py3-flask-markdown
+
+RUN apk add --no-cache python3-dev gcc musl-dev && \
```

Comment:
Hey. I just build this image and it was missing the py3-pip package.
Adding it in this line resolved the issue.

Reply:
That's curious, thanks for the report. Maybe something changed in the Alpine package. Mind send a PR?

Reply:
#260